src/transport/http/headers.ts

```diff
@@ -0,0 +1,90 @@
+import { ValidationError, CloudEvent } from "../..";
+import { headerMap as v1Map } from "./v1";
+import { headerMap as v03Map } from "./v03";
+import { Version } from "../../event";
+import { MappedParser } from "../../parsers";
+
+export interface Headers {
+  [key: string]: string
+};
+
+export const allowedContentTypes = ["application/json", "application/octet-stream"];
+export const requiredHeaders = [ "ce-id", "ce-source", "ce-type", "ce-specversion" ];
```

Comment:
We use the strings "ce-id", etc. many times.

Do we want to create a `v1/constants.ts` file or something equivalent and export constants with string from there?